lib/licenceDataClient.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const fetch = require('fetch-retry-or-die');
+const log = require('@financial-times/n-logger').default;
+const config = require('./helpers/config');
+const helpers = require('./helpers/helpers');
+
+const options = Object.assign({}, config.fetchOptions);
```

Comment:
Actually, that would mean that each time we call the `_getOptions()` function we would create a new object and add the `X-API-KEY` header (instead of just adding the `authorization` - if necessary).